docs/v1/use-cases.md

```diff
@@ -17,37 +17,121 @@ implement.
 would allow these operators to participate in the existing service broker
 ecosystem and grow their user base accordingly.
 
-## Low-Level Use Cases
+## CF Service Broker `v2` API Use Cases
 
-These are lower-level use cases the service catalog will implement in service
-to the high-level use cases.
+Initially, the catalog should support the current [CF Service Broker
+API](https://docs.cloudfoundry.org/services/api.html) These are the use cases
+that the service catalog has to implement in order to use that API.
 
-1.  As a service broker operator, I want to be able to register my broker with
+### Managing service brokers
+
+1.  As a catalog operator, I want a user to be able to register a broker with 
     the Kubernetes service catalog, so that the catalog is aware of the services
-    my broker offers
+    that broker offers
+2.  As a catalog operator, I want a user to be able to update a registered
+    broker so that the catalog can maintain the most recent versions of services
+    that broker offers
+3.  As a catalog operator, I want to be able to delete a broker from the 
+	catalog, so that I can keep the catalog clean of brokers I no longer want to
+	use
 
-### Registering a service broker with the catalog
+#### Registering a service broker with the catalog
 
 Each service broker must register with the service catalog to advertise the
 services it offers to the catalog.  After the broker has registered with the
 catalog, the catalog makes a call to the service broker's `/v2/catalog`
-endpoint.  The broker's response to this call is a list of services offered by
-that broker.
+endpoint.  The broker's returns a list of services offered by that broker.
+
+One of the attributes of a Service is a set of plans that differentiate the
+tiers of that service.  Services and plans both have an `id` attribute that
+is to uniquely identify instances of those resources in the catalog. 
+
+#### Updating a service broker
+
+Broker authors make changes to the services their brokers offer.  To refresh the
+services a broker offers, the catalog should re-list the `/v2/catalog` endpoint.
+The catalog should apply the result of re-listing the broker to its internal
+representation of that broker's services:
+
+1.  New service present in the re-list results are added
+2.  Existing services are updated if a diff is present
+3.  Existing services missing from the re-list are deleted
+
+TODO: spell out various update scenarios and how they affect end-users
+
+#### Delete a service broker
+
+There must be a way to delete brokers from the catalog.  In Cloud Foundry, it is
+possible to delete a broker and leave orphaned service instances.  We should
+evaluate where all broker deletes should:
+
+1.  Cascade down to the service instances for the broker
+2.  Leave orphaned service instances in the catalog
+3.  Fail if service instances still exist for the broker
+
+## CF Service Broker `v2` API Model
```

Comment:
Not sure we we're including this is a 'use case' doc. I agree with saying we should support it, but let's just point to the APIs themselves, otherwise we run the risk of duplicating things incorrectly and misleading people.